shared/search/user-pane/user.render.desktop.js

```diff
@@ -3,7 +3,7 @@
 import React, {Component} from 'react'
 import {Box, UserProofs, UserBio, UserActions} from '../../common-adapters'
 import {globalColors, globalStyles, globalMargins} from '../../styles'
-import {AVATAR_SIZE, HEADER_TOP_SPACE, HEADER_SIZE} from '../../profile/render.desktop'
+import {AVATAR_SIZE, HEADER_TOP_SPACE, HEADER_SIZE} from '../../profile/index.desktop'
```

Comment:
who not just use `../profile` instead of `../profile/index` ?

I'm just wondering because sometimes we get error messages with the filename of the import and it would be nicer if those said profile instead of just index.desktop (but maybe it would still say index.desktop.js anyways in which case it doesn't matter here)